ui/src/components/ConfigurationForm/ConfigurationForm.consts.ts

```diff
@@ -1,7 +1,7 @@
 import * as yup from 'yup';
 import { EffectType, EffectImplementation, JSONImplementation } from 'api/starter';
-import type { FormSelectOption } from '../FormSelect';
-import type { FormRadioOption } from '../FormRadioGroup';
+import type {FormSelectOption} from '../FormSelect';
```

Comment:
hm you and @lubarskyy should check on formatting - I think there's prettier in the project so this should be automatically applied by the IDE